ddm-lmp/src/main/java/de/hpi/ddm/messages/SourceByteMessage.java

```diff
@@ -0,0 +1,16 @@
+package de.hpi.ddm.messages;
+
+import akka.actor.ActorRef;
+import akka.stream.SourceRef;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class SourceByteMessage{
```

Comment:
- [ ] let's put this class into the LMP when we are finished so it all stays together